Multisig/UI/ClaimToken/ClaimSafeTokenFlow.swift

```diff
@@ -22,10 +22,15 @@ class ClaimSafeTokenFlow: UIFlow {
     }
 
     override func start() {
-        //TODO check claim availability
-        // if available show intro
-        showIntro()
-        // if not available show not available
+        //TODO remove workaround and check claim availability
+        if safe.addressValue == Address(exactly: "0xfF501B324DC6d78dC9F983f140B9211c3EdB4dc7") {
```

Comment:
do I understand your intent right that setting it unavailable for demo safe is because we want to test it for other safes just fine?

Reply:
it's about making `claim unavailable` screen accessible for testing. Due to missing availability check flow always starts as if claim is available. Demo safe will show `claim unavailable` screen for now.